tests/test_uninstall.py

```diff
@@ -0,0 +1,405 @@
+import os
+import sys
+from pathlib import Path
+from shutil import rmtree
+from subprocess import SubprocessError
+
+import pytest
+from conda.base.constants import COMPATIBLE_SHELLS
+from conda.common.path import win_path_to_unix
+from conda.core.initialize import (
+    _read_windows_registry,
+    make_initialize_plan,
+    run_plan,
+    run_plan_elevated,
+)
+from conftest import _get_shortcut_dirs, menuinst_pkg_specs, run_conda
+from ruamel.yaml import YAML
+
+ON_WIN = sys.platform == "win32"
+ON_MAC = sys.platform == "darwin"
+ON_LINUX = not (ON_WIN or ON_MAC)
+ON_CI = bool(os.environ.get("CI"))
```

Comment:
This will be true for `CI=0` 🤔

Reply:
True, I improved that check. We have a similar thing in `constructor` - I will port it there, too.